specification/templates/CONTRIBUTING.md

```diff
@@ -1,43 +1,51 @@
 # Contributing Guidelines
 
 Thank you for your interest in contributing to our repository! Whether it's a bug
-report, new feature, question, or additional documentation, we greatly value
-feedback and contributions from our community. Read through this document
-before submitting any issues or pull requests to ensure we have all the
-necessary information to effectively respond to your bug report or
-contribution.
+report, new feature, or question, we greatly value feedback and contributions from
+our community. Read through this document before submitting any issues or pull 
+requests to ensure we have all the necessary information to effectively respond
+to your bug report or contribution.
 
-In addition to this document, please review our [Code of
-Conduct](CODE_OF_CONDUCT.md). For any code of conduct questions or comments
-please email oss@splunk.com.
+In addition to this document, review our [Code of Conduct](CODE_OF_CONDUCT.md). 
+For any code of conduct questions or comments, send an email to oss@splunk.com.
 
 ## Reporting Bugs/Feature Requests
 
 We welcome you to use the GitHub issue tracker to report bugs or suggest
-features. When filing an issue, please check existing open, or recently closed,
-issues to make sure somebody else hasn't already reported the issue. Please try
-to include as much information as you can. Details like these are incredibly
-useful:
+features. When filing an issue, check existing open, or recently closed,
+issues to make sure somebody else hasn't already reported the issue. Try
+to include as much information as you can. Details like these can be useful:
 
 - A reproducible test case or series of steps
 - The version of our code being used
 - Any modifications you've made relevant to the bug
 - Anything unusual about your environment or deployment
 - Any known workarounds
 
-When filing an issue, please do *NOT* include:
+When filing an issue, do *NOT* include:
 
-- Internal identifiers such as JIRA tickets
+- Internal identifiers such as Jira tickets
 - Any sensitive information related to your environment, users, etc.
 
 ## Reporting Security Issues
 
-See [SECURITY.md](SECURITY.md#reporting-security-issues) for detailed instructions.
+See [SECURITY.md](SECURITY.md#reporting-security-issues) for instructions.
+
+## Documentation
+
+The Splunk Observability documentation is hosted on https://docs.splunk.com/Observability,
+which contains all the prescriptive guidance for Splunk Observability products. 
+Prescriptive guidance consists of step-by-step instructions, conceptual material,
+and decision support for customers. Reference documentation and development 
+documentation is hosted on this repository.
```

Comment:
should it not be in the main `README.md` ?

Reply:
Possibly, but we don't have a README template. :)